mced.h

```diff
@@ -119,7 +121,7 @@ struct mce {
 	uint8_t  bank;		/* MC bank */
 	int8_t   vendor;	/* CPU vendor (enum cpu_vendor) */
 } __attribute__ ((packed));
-#define MCE_STRUCT_VER	1
+#define MCE_STRUCT_VER	2
```

Comment:
AFAICT we don't use this anywhere?

Reply:
Correct. I'll remove it in a follow up patch.